samples/net/common/sample_app_setup.c

```diff
@@ -0,0 +1,222 @@
+/* sample_app_setup.c */
+
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if 1
```

Comment:
Enabling debug by default for sample apps is currently done on purpose. Because there is no kconfig for the sample apps debug options, it makes sense to activate the debugging by default for them.